code/datums/martial/cqc.dm

```diff
@@ -221,7 +221,9 @@
 			to_chat(A, span_danger("You strike [D]'s jaw, leaving [D.p_them()] disoriented!"))
 			playsound(get_turf(D), 'sound/weapons/cqchit1.ogg', 50, TRUE, -1)
 			if(I && D.temporarilyRemoveItemFromInventory(I))
```

Comment:
``temporarilyRemoveItemFromInventory`` checks for NODROP trait and returns false if it has it, why is this working?

Reply:
Your issue report link says nozzles, I read the files changed and saw the comment, then thought this was about NODROP.

nozzles' item flags are ``NOBLUDGEON | ABSTRACT``, I think they should be working with ``DROPDEL`` to snap back in place when you drop it instead, or can just be given nodrop trait?

Reply:
this was an attempt to blanket fix all items of such nature like nozzles, defib paddles, maybe the gatling laser and other stuff I don't remember, because this issue will be present on them as well, probably. 
not quite sure how to go about it better either, and i think offering and putting an item like this in other hands is also possible without CQC involved. maybe a stricter setting for those items so they can't leave their parent item's wearer at all times? most of them check for being on a mob, rather than on a parent item's holder exclusively